example/server.js

```diff
@@ -0,0 +1,19 @@
+var hapi = require('hapi');
+var home = require('./home');
```

Comment:
not a fan of writing plugins for endpoints inside the app.
prefer to use a `routes.js` file where _all_ routes are listed alphabetically with corresponding 
`require('../handlers/descriptive-route-name.js')` ...
to be clear, we should _absolutely_ use plugins for specific _functionality_, and that functionality should be split out into a _separate independently tested & documented_ module/package which can be included into the project. This will reduce the amount of code in the _main_ project while hopefully improving the re-usability of selected bits of our code.
e.g `hapi-login` which is meant to _independently_ solve the "login" functionality.